Makefile.am

```diff
@@ -39,6 +40,7 @@ rdmainclude_HEADERS =
 
 # internal utility functions shared by in-tree providers:
 common_srcs =				\
+	src/hmem.c			\
```

Comment:
I believe I need to also add "include/ofi_hmem.h" to the src_libfabric_la_SOURCES section.